web-ui/src/main/java/org/vaadin/example/queryview/WhereTextBoxesLayout.java

```diff
@@ -54,13 +54,27 @@ public WhereTextBoxesLayout() {
                 });
         pauseButton.addClickListener(
                 e -> {
-                    isEnabled = !isEnabled;
+                    pauseRow();
                 });
+        pauseButton.getElement().setProperty("title", "Pauses (Disables) this row of queries");
```

Comment:
Maybe extract this string into a variable so it is guaranteed to be equal to the string in `pauseRow()`. Also, "queries" is not the correct term here. Instead, I would only speak of "row" or "triplet" maybe with the pre-/postfix "subject, predicate, and object" to make it clear.

Reply:
Good ideas. I've implemented the changes and pushed the commit

Reply:
I am getting a little pedantic but I think there is a "d" missing in `tooltipMessageDisable` and `tooltipMessageEnable`. Also, you speak of a "row of triplets" when one row only forms one triplet. Maybe we should avoid the term "triplet" altogether as it might be confusing to users. Instead, I would go with "Disables this row of subject, predicate, and object."